sites/docs/pages/components/point-map.md

```diff
@@ -0,0 +1,455 @@
+    ]}
+/>
+```
+
+#### With clickable link and `tooltipType=click`
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    value=sales 
+    valueFmt=usd 
+    pointName=point_name 
+    height=200
+    tooltipType=click
+    tooltip={[
+        {id: 'point_name', showColumnName: false, valueClass: 'text-xl font-semibold'},
+        {id: 'sales', fmt: 'eur', fieldClass: 'text-[grey]', valueClass: 'text-[green]'},
+        {id: 'link_col', showColumnName: false, contentType: 'link', linkLabel: 'Click here', valueClass: 'font-bold mt-1'}
+    ]}
+/>
+
+```svelte
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    value=sales 
+    valueFmt=usd 
+    pointName=point_name 
+    height=200
+    tooltipType=click
+    tooltip={[
+        {id: 'point_name', showColumnName: false, valueClass: 'text-xl font-semibold'},
+        {id: 'sales', fmt: 'eur', fieldClass: 'text-[grey]', valueClass: 'text-[green]'},
+        {id: 'link_col', showColumnName: false, contentType: 'link', linkLabel: 'Click here', valueClass: 'font-bold mt-1'}
+    ]}
+/>
+```
+
+### Custom Color Palette
+
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    value=sales 
+    valueFmt=usd 
+    pointName=point_name 
+    height=200
+    colorPalette={['yellow','orange','red','darkred']}
+/>
+
+```svelte
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    value=sales 
+    valueFmt=usd 
+    pointName=point_name 
+    height=200
+    colorPalette={['yellow','orange','red','darkred']}
+/>
+```
+
+### Custom Styling
+
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    pointName=point_name 
+    height=200
+    color=#128c2b
+    size=10
+    opacity=0.6
+    borderWidth=0
+/>
+
+```svelte
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    pointName=point_name 
+    height=200
+    color=#128c2b
+    size=10
+    opacity=0.6
+    borderWidth=0
+/>
+```
+
+### Link Drilldown
+Pass in a `link` column to enable navigation on click of the point. These can be absolute or relative URLs
+
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    link=link_col 
+    height=200
+/>
+
+```svelte
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    link=link_col 
+    height=200
+/>
+```
+
+### Use Map as Input
+Use the `name` prop to set an input name for the map - when a point is clicked, it will set the input value to that row of data
+
+```svelte
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    name=my_point_map 
+    height=200
+/>
+```
+
+<PointMap 
+    data={la_locations} 
+    lat=lat 
+    long=long 
+    name=my_point_map 
+    height=200
+/>
+
+*Click a point on the map to see the input value get updated:*
+
+#### Selected value for `{inputs.my_point_map}`: 
+  
+{inputs.my_point_map}
+
```

Comment:
Think this is leftover code -- just renders [object Object]

Reply:
Unfortunately this is the default selection for the input at the moment - since that line doesn't reference a specific property it shows the object. though JSON.stringify shows undefined strangely.

If you select an item on the map it should populate the object with the columns from that row

Reply:
Ah, I didn't get that it included the whole row from the example. Maybe a couple additional column references would be more clear than the object? 

![CleanShot 2024-05-17 at 11 38 06](https://github.com/evidence-dev/evidence/assets/6857673/5f82d05a-d048-49f5-86b7-76c8dcf682aa)

Reply:
thanks - this is different than what i was seeing before, but fixed now thanks to @ItsMeBrianD!
![CleanShot 2024-05-17 at 13 12 30](https://github.com/evidence-dev/evidence/assets/12602440/0242c071-2b78-4a35-a9a4-598478e9ac61)